rosserial_cortex/.gitignore

```diff
@@ -0,0 +1,2 @@
+./working-rosserial-client/include/ros_lib/*
```

Comment:
These shouldn't be necessary. We should encourage people to work outside the source directory if possible.